nodes.py

```diff
@@ -1952,7 +1961,16 @@ def load_custom_nodes():
             logging.info("{:6.1f} seconds{}: {}".format(n[0], import_message, n[1]))
         logging.info("")
 
-def init_custom_nodes():
+def init_builtin_custom_nodes():
```

Comment:
This should be called `init_builtin_extra_nodes`

Reply:
Done.